### ISSUE TITLE
Analog slider very slow

### ISSUE THREAD

**Issue by EwanGreen4** [romgrk/web-toolkit#16]
Not sure exactly what the widget/UI element is called, because as you can see I've never used Adwaita and don't even know the terminology. 
![image](https://user-images.githubusercontent.com/36348486/106936439-cd1b8000-66d9-11eb-8876-4bfcb73382a5.png)
This slider here is very slow to update in the web preview. Don't know if it's an implementation issue, or a problem with the slider itself, but I would assume the latter because the value changing box thing on row 3 updates just fine when I move the slider. Because it updates faster than the slider, I can say with confidence it's probably just a graphical issue. Sorry I couldn't be of more help. 
Firefox 84.0.2, Arch Linux.

**Comment by romgrk** [comment]
Yes I know about that, I've re-used the slider from material-ui but there is a performance issue that I haven't had the time to fix yet.

This is a duplicate of #5 but I'll keep this one open as the name is better.
